backend/app/helpers/spree/admin/stock_locations_helper.rb

```diff
@@ -4,9 +4,7 @@ module Spree
   module Admin
     module StockLocationsHelper
       def admin_stock_location_display_name(stock_location)
-        name_parts = [stock_location.admin_name, stock_location.name]
-        name_parts.delete_if(&:blank?)
-        name_parts.join(' / ')
+        [stock_location.admin_name, stock_location.name].compact.join(' / ')
```

Comment:
`.compact` and `delete_if(&:blank?)` don't do the same thing. `.compact` only removes `nil` values, not empty strings.

```suggestion
        [stock_location.admin_name, stock_location.name].reject(&:blank?).join(' / ')
```